test/test_grocy.py

```diff
@@ -24,7 +25,7 @@ def test_get_chores_valid(self):
         chores = self.grocy.chores(get_details=True)
         
         assert isinstance(chores, list)
-        assert len(chores) == 6
+        self.assertEqual(len(chores), 6)
         assert chores[0].id == 1
```

Comment:
I guess switching all assertions to the `self` equivalent would make sense?

Reply:
Not sure because others tests are >= not ==

Reply:
There are several assertions available: https://docs.python.org/3/library/unittest.html#unittest.TestCase.debug

Stuff like `>=` would probably get wrapped in an `assertTrue`

Reply:
Also we're providing or own test data, so specific assertions using equals should be possible

Reply:
Any + value ?
Because I'm losing the main goal of this PR.

Reply:
This could also be done in a separate PR, I just noticed because that line got changed. I'll create an issue for this